lib/shrinkwrap-package.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var Package = require('./package');
+
+function ShrinkwrapPackage(name, versionSpecified, versionInstalled, parents) {
+  this._super$init.call(this, name, versionSpecified, versionInstalled);
+  this.init(name, versionSpecified, versionInstalled, parents);
+}
+
+ShrinkwrapPackage.prototype = Object.create(Package.prototype);
+ShrinkwrapPackage.prototype.constructor = ShrinkwrapPackage;
+
+ShrinkwrapPackage.prototype._super$init = Package.prototype.init;
+ShrinkwrapPackage.prototype.init = function(name, versionSpecified, versionInstalled, parents) {
+  this.parents = parents;
+};
+
+ShrinkwrapPackage.prototype.updateRequired = function() {
+  return this.versionSpecified !== this.versionInstalled;
```

Comment:
@mixonic Do you think we should this `updateRequired` function with this https://github.com/quaertym/ember-cli-dependency-checker/blob/master/lib/package.js#L18-19?